### PR TITLE
Add some missing parts of git docs

### PR DESCRIPTION
**Pull request by mmorhun**
This PR adds docs for:
- git commit window
- git push dialog
- git pull dialog
- git branches window
- git compare window

### CODE REVIEW

File: src/main/_docs/ide/ide-git-svn.md [hunk 1]
```diff
@@ -184,14 +191,84 @@ Files in project explorer and editor tabs can be colored according to their Git
 * Blue: files that contain changes.
 * Yellow: files that are not staged in index.
 
-## Reviewing changes with the diff window
-With the diff window it is possible to review changes between two states of code. To view the diff, use `Git`->`Compare`->`<Select-to-what>` from main menu. If more than one file has changed the list of changed files will be opened first. To select a file to compare double-click on it or select a file then click the `Compare` button.
+## Committing changes
+To commit your changes use the git commit window. To show it click `Git` -> `Commit...` from main menu or just use `Alt + c` shortcut.
```
Comment (vinokurig) [inline src/main/_docs/ide/ide-git-svn.md]: Please note that it is also possible to do that from project explorer's context menu and from editor's context menu

File: src/main/_docs/ide/ide-git-svn.md [hunk 2]
```diff
@@ -184,14 +191,84 @@ Files in project explorer and editor tabs can be colored according to their Git
...
+After this, hit `Pull` button to proceed.
+
+## Managing branches
+To manage your git branches use branches window.
+To open it use `Git` -> `Branches...` from main menu or `Ctrl + b` shortcut.
+
+![git-branches-window.png]({{base}}{{site.links["git-branches-window.png"]}}){:style="width: auto"}
+
+In the middle of the window is placed a list of all branches in current project.
+It is possible to display only local or only remote branches by `Show branches` drop-down above.
```
Comment (vinokurig) [inline src/main/_docs/ide/ide-git-svn.md]: Please note that it is also possible to display both remote and local in one list

Comment (mmorhun) [reply src/main/_docs/ide/ide-git-svn.md]: Yes, this is default behaviour. Here is said, that a user can filter branches in this way

File: src/main/_docs/ide/ide-git-svn.md (hunk 1 above)
Comment (mmorhun) [reply src/main/_docs/ide/ide-git-svn.md]: yes, thank you, this must be documented.

**Pull request merged**